ui/components/imageWorkspace/src/lib/api/objectsApi.ts

```diff
@@ -156,9 +156,9 @@ export const sortAndFilterObjectsToAnnotate = (
       return confidence >= confidenceFilterValue[0];
     })
     .sort((a, b) => {
-      const confidenceA = a.bbox?.confidence || 0;
-      const confidenceB = b.bbox?.confidence || 0;
-      return confidenceB - confidenceA;
+      const confidenceA = a.bbox?.coords[0] || 0;
+      const confidenceB = b.bbox?.coords[0] || 0;
+      return confidenceA - confidenceB;
     });
```

Comment:
"confidenceA/B" var name is not relevant anymore. But maybe both sort should be available later ?

Reply:
I've rename the var for now. 
Allowing both sorts would have quite big impact on the UI so we decided to keep that feature for later. It would be difficult then to redo this sort function !